modules/cql/test/blaze/elm/literal.clj

```diff
@@ -92,6 +92,20 @@
     (assoc :unit unit)))
 
 
+;; 3.10. Ratio
+(defn ratio [[[numerator-value numerator-unit] [denominator-value denominator-unit]]]
```

Comment:
Please add a newline before the argument vector if it doesn't fit in a line width of 80 chars.